docs/docs/tour/tour.md

```diff
@@ -0,0 +1,152 @@
+# Tour
+
+Once you've started the DBSP demo, as described in [Get
+Started](../intro), navigate to [localhost:8085](http://localhost:8085)
+in your web browser to view the DBSP user interface.  A status
+dashboard fills most of the home screen, as shown below, with a
+sidebar on the left offering access to tabs for DBSP features:
+
+![DBSP home screen](home.png)
+
+Let's take a quick tour around the options on the sidebar.
+
+## Analytics
+
+![Analytics](analytics.png)
+
+The top group of tabs allows a DBSP user to define the structure of
+data and specify analyses.  Both are these are written as SQL: data
+definitions as `CREATE TABLE` commands, and analyses as `CREATE VIEW`
+commands.  DBSP implements a rich SQL feature set that permits
+sophisticated SQL written for databases such as Postgres and MySQL to
+run with only minimal changes.
+
+The "SQL Editor" tab allows a user to add and name new SQL programs by
+typing them or pasting them in.
+
+Click on "Existing Programs" to show a list of the SQL programs that
+are already defined.  This list displays each program's name and
+description, which one may double-click to edit.  Each program also
+has a status that indicates whether it has been fully compiled, and
+two icons for actions: a pencil ![pencil](pencil.png) to view or edit
+the program's SQL, and a trash can ![trash can](trash.png) to delete
+it:
+
+![Existing Programs](existing-programs.png)
+
+Click on the pencil icon for one of the rows.  This brings up the SQL
+Editor for the selected program, which allows one to edit the
+program's name and description and code:
+
+![SQL Editor](sql-editor.png)
+
+Scroll through the SQL code.  You should see that it consists of
+`CREATE TABLE` and `CREATE VIEW` commands:
+
+* `CREATE TABLE` specifies a DBSP input stream.  For DBSP, which
```

Comment:
I would argue that it's not a misnomer. What happens is that we treat a table as a stream of changes, and we compute every time the table changes, unlike a database. In databases computations are ephemeral, and are executed for a certain version of the tables. We execute the computation for *all* versions of all tables. But they are still tables.